reconcile/test/test_slack_usergroups.py

```diff
@@ -86,6 +89,35 @@ def test_get_pagerduty_username_slack_username(self):
         result = integ.get_pagerduty_name(user)
         self.assertEqual(result, 'pd')
 
+    @patch.object(integ, 'SlackApi')
+    @patch.object(queries, 'get_app_interface_settings', autospec=True)
+    @patch.object(queries, 'get_permissions')
+    def test_get_slack_map_return_expected(self, get_permissions,
+                                           get_app_interface_settings,
+                                           slackApi):
+        get_permissions.return_value = self.get_permissions_fixture()
+        slack_api_mock = create_autospec(SlackApi)
+        expected_slack_map = {
+            'coreos': {
+                'slack': slack_api_mock,
+                'managed_usergroups': [
+                    'app-sre-team',
+                    'app-sre-ic'
+                ]
+            }
+        }
+        result = integ.get_slack_map()
+        slackApi.assert_called_once()
+        self.assertEqual(result['coreos']['managed_usergroups'],
+                         expected_slack_map['coreos']['managed_usergroups'])
+        self.assertIn('slack', result['coreos'].keys())
```

Comment:
Nit: You don't need the `keys` argument here, which in the event of bugs will give a more detailed ourput:
```suggestion
        self.assertIn('slack', result['coreos'])
```